docs/topics/GATEWAY.md

```diff
@@ -89,6 +89,7 @@ Used to trigger the initial handshake with the gateway.
 | properties | object | connection properties |
 | compress | boolean | whether this connection supports compression of the initial ready packet |
 | large_threshold | integer | value between 50 and 250, total number of members where the gateway will stop sending offline members in the guild member list |
+| shard | array of two integers (shard_id, num_shards) | used for [Guild Sharding](#DOCS_GATEWAY/sharding) |
```

Comment:
Isn't the shard key a tuple when using ETF? Maybe this should be clarified here.